mlx/ops.h

```diff
@@ -256,30 +256,36 @@ array moveaxis(
     int destination,
     StreamOrDevice s = {});
 
+enum PaddingMode { Constant, Edge };
```

Comment:
Maybe use an `enum class` here instead of a `enum`

Reply:
On the other hand we tend to match types from Python API to C++ API (I'm not sure that's the best choice tbh), but for the sake of consistency maybe simpler to have it be a string at the `ops.h` level and scope the enum to the Primitive itself.

I think (?) it also makes it easier for code binding ops.h (i.e. MLX C) to not have to add the new enum type.

E.g. like `indexing` for `meshgrid`, `ord` for `norm`.

Reply:
Sounds good! I changed it to just be a string.